Content.Client/EntryPoint.cs

```diff
@@ -144,7 +144,10 @@ public override void Init()
                 "Mop",
                 "Bucket",
                 "Puddle",
-                "CanSpill"
+                "CanSpill",
+                "RandomPottedPlant",
+                "CommunicationsConsole",
+                "BarSign"
```

Comment:
Does C# allow for trailing commas in array initializers? If so, it would be a good addition to prevent extra line edits.

Reply:
Yeah it does.
```suggestion
                "BarSign,"
```